src/main/java/org/cyclonedx/gradle/utils/CycloneDxUtils.java

```diff
@@ -71,6 +71,7 @@ public static void writeBom(
             final Version version,
             final String formats) {
 
+        sortBom(bom);
```

Comment:
Components and dependencies in the BOM are already sorted by using TreeSet here https://github.com/CycloneDX/cyclonedx-gradle-plugin/blob/ba5369b4f3792b9fcaf3237f26f75b0cd7014c53/src/main/java/org/cyclonedx/gradle/SbomBuilder.java#L95-L96. 

If you want to change the order, it must be done [here for components](https://github.com/CycloneDX/cyclonedx-gradle-plugin/blob/ba5369b4f3792b9fcaf3237f26f75b0cd7014c53/src/main/java/org/cyclonedx/gradle/model/ComponentComparator.java) and [here for dependencies](https://github.com/CycloneDX/cyclonedx-gradle-plugin/blob/ba5369b4f3792b9fcaf3237f26f75b0cd7014c53/src/main/java/org/cyclonedx/gradle/model/DependencyComparator.java)